src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
Ensure that the number of `div.calendar__day` elements matches the `calendar--month-length-xx` modifier. If 'xx' is 31, there should be exactly 31 of these elements.